features/csvw.feature

```
@@ -121,3 +121,167 @@ Feature: Create CSVW metadata
     When I create a CSVW file from the mapping and CSV
     Then the metadata is valid JSON-LD
     And the input format of the csv is recorded as csv
+
+  Scenario: CSVMetadata from mapping with measure type and unit columns
+    Given a CSV file 'observations.csv'
+      | Period          | Flow    | HMRC Reporter Region | HMRC Partner Geography | SITC 4 | Value | Measure Type | Unit          |
+      | quarter/2018-Q1 | exports | EA                   | A                      | 01     | 2430  | net-mass     | kg-thousands  |
+      | quarter/2018-Q1 | exports | EA                   | A                      | 02     | 2     | net-mass     | kg-thousands  |
+      | quarter/2018-Q4 | imports | ZB                   | TR                     | 88     | 10    | gbp-total    | gbp-thousands |
+      | quarter/2018-Q4 | imports | ZB                   | TR                     | 89     | 352   | gbp-total    | gbp-thousands |
```

Shouldn't the measure types be `net mass` and `monetary value` (or similar)?

---

Yes. I was just basing this off some existing Tidy data and avoiding changing things as it's not part of this issue. However, I agree that it's distracting especially if this example ends up being canonical: we don't want to propagate the errors.